strapi/strapi
# [RTE] Fix blocks config (list, sort, filter, main field)

### What does it do?

- Adds blocks to the list of attributes that should _not_ be listable, sortable, or filterable

### Why is it needed?

Blocks should behave like JSON / richtext

### How to test it?

- Blocks should not be an option as a main field in configure the view
- Blocks should not show up in the list view and should not be an option to display in the list view
	- Blocks should not be sortable
	- Blocks should not be filterable or show up in the filter options


Fixes #18203
